apps/web/composables/useLocalization/useLocalization.ts

```diff
@@ -101,15 +101,18 @@ export const useLocalization = createSharedComposable(() => {
    * @description Function for switching app locale.
    * @param language
    *
+   * @param hideMenu
    * @example switchLocale('en')
    */
-  const switchLocale = async (language: string) => {
+  const switchLocale = async (language: string, hideMenu = true) => {
```

Comment:
**question**: What's the point of `hideMenu`? It doesn't get used as far as I can tell.

Reply:
We need it here to fix a visual glitch.